### DISCUSSION TITLE
Maplibre - What is the recommended procedure for including the CSS of a fork.

### DISCUSSION THREAD

**Discussion by astridx** [uber/react-map-gl#1498] [Q&A]
I am new with [MapLibre](https://github.com/MapLibre) and react-map-gl. I have configured everything as described in the [documentation](https://visgl.github.io/react-map-gl/docs/get-started/get-started#using-with-a-mapbox-gl-fork). Almost everything works fine. 

The only thing I would like to change at the moment. The following message appears in the browser console and I have to include the MapLibre Css file manually. 

```
GEThttps://api.tiles.mapbox.com/mapbox-gl-js/v1.14.0/mapbox-gl.css
[HTTP/1.1 404 Not Found 714ms]
```

I think it is because of this: https://github.com/visgl/react-map-gl/blob/227afdbdcd47d8c036ec6ba105662171c53a8a9b/src/mapbox/mapbox.js#L422. This URL does not exist. Edit: For MapLibre this should be something like: `https://unpkg.com/maplibre-gl@1.14.0-rc.1/dist/maplibre-gl.css`

Now I would like to know if I am doing something wrong. What is the recommended procedure for including the CSS of a fork.

Here is my repo: 
https://github.com/astridx/maplibre-app/tree/1_maplibre-map

**Comment by Pessimistress** [comment]
You need to manually include the CSS in your page, as described in https://visgl.github.io/react-map-gl/docs/get-started/get-started#styling. The default fallback points to `mapbox.com` because there is no way for this library to tell whether you are using a fork.
